### ISSUE TITLE
[FEATURE] can someone help me integrate a main layout with attendize?

### ISSUE THREAD

**Issue by levycamara** [Attendize/Attendize#1018]
I have a front-end project for a website with several events, would it be possible for someone to integrate this layout with the attendize platform? If so, what would be the budget for this?

Grateful.

**Comment by justynpride** [comment]
@levycamara Did your front project happen? I’d be interested in know how it went etc..

**Comment by modifiedcontent** [comment]
Yes, I'm also struggling to integrate Attendize in existing sites and customize the whole registration process, since the embed widget doesn't seem to work anymore and the application's default event page etc. is awful.

If anyone has figured it out, please share here. 

I've made edits /assets/stylesheet/frontend.css, but they are apparently ignored, so I guess the application uses caching or there is some clever compiling going on elsewhere that is over my head?

Ideally I would be able to include just the registration form into my existing event sites, without all the crap about organiser, a weirdly formatting map for "location", etc.
